indico/modules/events/abstracts/templates/management/abstract_list.html

```diff
@@ -73,15 +73,32 @@
                     </ul>
                 </div>
                 <div class="group">
-                    <a class="i-button icon-checkmark js-requires-selected-row js-dialog-action accept">
-                        {% trans %}Accept{% endtrans %}
+                    <a class="i-button icon-checkmark accept js-requires-selected-row"
+                       data-href="{{ url_for('.manage_judge_abstracts', event) }}"
+                       data-title="{% trans %}Accept selected abstracts{% endtrans %}"
+                       data-params-selector="#abstract-list input[name=abstract_id]:checked"
+                       data-params='{"action": "{{ actions.accept.name }}"}'
```

Comment:
Just a clarification. The reason not to go through the `enum` is that if we ever change the class, we would need to change it here as well. OTOH, accessing it as `accept.name` would make our lives much easier in case of such refactoring, as searching just `accept` would return countless unrelated results.